### PR TITLE
Issue/196/map refresh timer

### LINKED ISSUE #196: Map refresh timer
We have to add timer (maybe above map or in menu) for how long the map will be refresh.

### PR DESCRIPTION
**Pull request by Bedpaw**
Closes #196

### CODE REVIEW

File: src/components/molecules/countdown-timer.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<template>
+  <div :style= "[minutes === 0 ? {color: 'red'} : {color: 'black'}]">
```
Comment (dbetka) [inline src/components/molecules/countdown-timer.vue:2]: It should be in CSS classes - create 2 classes for 2 states and use these here.

File: src/components/molecules/countdown-timer.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    {{getMinutes}}:{{getSeconds}}
+  </div>
+</template>
+
+<script>
+import { getZeroPad } from 'utils/date';
```
Comment (dbetka) [inline src/components/molecules/countdown-timer.vue:8]: What getZeroPad method doing? Name is not clear

File: src/components/molecules/countdown-timer.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      if (this.seconds < 0) return '00';
+      return getZeroPad(this.seconds);
+    },
+    getMinutes () {
+      return getZeroPad(this.minutes);
+    },
+  },
+  methods: {
+    updateDateTime () {
+      if (this.seconds === 0) {
```
Comment (dbetka) [inline src/components/molecules/countdown-timer.vue:35]: Check moment.js plugin to manage time easier - https://momentjs.com/

Comment (dbetka) [reply src/components/molecules/countdown-timer.vue:35]: You can add seconds and plugin will update minutes by itself.

**Review by dbetka — CHANGES_REQUESTED** [review]

File: src/dictionary/language/pl.js [hunk 4]
```diff
@@ -158,4 +158,9 @@ export const pl = {
       success: 'Twoje hasło zostało zmienione!',
     },
   },
+  features: {
+    timer: {
+      toUpdate: 'Aktualizacja',
+    },
+  }
```
Comment (dbetka) [inline src/dictionary/language/pl.js:165]: A comma is missing.

**Review by dbetka — CHANGES_REQUESTED** [review]

File: src/components/molecules/countdown-timer.vue (hunk 3 above)
Comment (dbetka) [reply src/components/molecules/countdown-timer.vue:35]: Maybe not momentjs

File: src/dictionary/language/pl.js (hunk 4 above)
Comment (Bedpaw) [reply src/dictionary/language/pl.js:165]: done

File: src/components/molecules/countdown-timer.vue (hunk 2 above)
Comment (Bedpaw) [reply src/components/molecules/countdown-timer.vue:8]: It is adding zero "padding" to time format, it's popular name for this. idk if we should change it

File: src/components/molecules/countdown-timer.vue (hunk 1 above)
Comment (Bedpaw) [reply src/components/molecules/countdown-timer.vue:2]: done

File: src/components/molecules/countdown-timer.vue (hunk 2 above)
Comment (dbetka) [reply src/components/molecules/countdown-timer.vue:8]: OK, approve

File: src/components/molecules/countdown-timer.vue (hunk 3 above)
Comment (dbetka) [reply src/components/molecules/countdown-timer.vue:41]: Create issue "Change moment.js to a different library" and close conversation

Comment (Bedpaw) [reply src/components/molecules/countdown-timer.vue:41]: done

**Pull request merged**